test/checks/missing_enable_comment_test.rb

```diff
@@ -90,4 +90,78 @@ def test_specific_checks_disabled_and_reenabled_with_all_checks_disabled
       All checks were disabled but not re-enabled with theme-check-enable at templates/index.liquid
     END
   end
+
+  def test_adds_missing_enable_all
+    expected_sources = {
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable{% endcomment %}
+        {% assign x = 1 %}
+        {% comment %}theme-check-enable{% endcomment %}
+      END
+    }
+    sources = fix_theme(
+      ThemeCheck::MissingEnableComment.new,
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable{% endcomment %}
+        {% assign x = 1 %}
+      END
+    )
+    sources.each do |path, source|
+      assert_equal(expected_sources[path], source)
+    end
+  end
+
+  def test_adds_missing_enable_specific_checks
+    expected_sources = {
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable TracerCheck, AnotherCheck{% endcomment %}
+        {% assign x = 1 %}
+        {% comment %}theme-check-enable TracerCheck, AnotherCheck{% endcomment %}
+      END
+    }
+    sources = fix_theme(
+      ThemeCheck::MissingEnableComment.new,
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable TracerCheck, AnotherCheck{% endcomment %}
+        {% assign x = 1 %}
+      END
+    )
+    sources.each do |path, source|
+      assert_equal(expected_sources[path], source)
+    end
+  end
+
+  def test_adds_missing_enable_after_last_check_when_multiple_checks_disabled
+    expected_sources = {
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable OneCheck, TwoCheck{% endcomment %}
+          {% assign x = 1 %}
+        {% comment %}theme-check-disable TracerCheck, AnotherCheck{% endcomment %}
+          {% assign x = 1 %}
+        {% comment %}theme-check-disable{% endcomment %}
+          {% assign x = 1 %}
+        {% comment %}theme-check-enable{% endcomment %}
+      END
+    }
+    sources = fix_theme(
+      ThemeCheck::MissingEnableComment.new,
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable OneCheck, TwoCheck{% endcomment %}
+          {% assign x = 1 %}
+        {% comment %}theme-check-disable TracerCheck, AnotherCheck{% endcomment %}
+          {% assign x = 1 %}
+        {% comment %}theme-check-disable{% endcomment %}
```

Comment:
Will it work if I remove this line?

By work I mean, either not correct it at all of still append the proper `{% comment %}theme-check-enable{% endcomment %}`. It should leave the template in an acceptable state.